Tiltfile.dev

```diff
@@ -0,0 +1,17 @@
+local_resource(
+  name = 'dev:acsp-api',
+  cmd = 'mvn clean compile',
+  deps = ['src']
+)
+custom_build(
+  ref = '169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/acsp-api',
```

Comment:
Not sure if this line should be changed to reflect the correct region. Can you find out if this should be changed to be 

`eu-west-2`

Reply:
updated to '416670754337.dkr.ecr.eu-west-2.amazonaws.com/acsp-api'